wallet/src/main/java/com/android/identity_credential/wallet/ui/CommonUI.kt

```diff
@@ -127,6 +129,27 @@ fun KeyValuePairText(
     }
 }
 
+@Composable
+fun KeyValuePairHtml(
+    keyText: String,
+    html: String
+) {
+    Column(
+        Modifier
+            .padding(8.dp)
+            .fillMaxWidth()) {
+        Text(
+            text = keyText,
+            fontWeight = FontWeight.Bold,
+            style = MaterialTheme.typography.titleMedium
+        )
+        Text(
+            text = AnnotatedString.Companion.fromHtml(html),
```

Comment:
Please add a note since we need to revisit this when porting to multi-platform: `// TODO: The fromHtml() extension is only available on Android.`

Reply:
Added.